cvpy/tests/test_imagetable.py

```diff
@@ -266,6 +266,139 @@ def test_imagetable_from_table_custom_col_names(self):
+        path = "images/"
+
+        # If path exists, we are on the Jenkins server
+        if exists(path):
+            # Images are determined to be similar if the average pixel value of all channels
+            # are less than three percent of their maximum value.
+            validThreshold = 255 * 0.03
+
+            # Test loading images fron both client and server. One will have a valid path and
+            # return value to test the type, the other will try to load from a nonexistent directory
+            # and the last will load images from the server
+            natural = ImageTable.load_client_images(
+                output_table_parms={'name': 'imgsClientDirectory', 'caslib': 'CASUSER(user)'},
+                path=path,
+                connection=self.s,
+                subdirs=False
+            )
+            ImageTable.load_client_images(
+                output_table_parms={'name': 'imgsClientEmpty', 'caslib': 'CASUSER(user)'},
+                path='path/does/not/exist',
+                connection=self.s,
+                subdirs=False
+            )
+            self.s.CASTable('imgsServer', replace=True)
+            self.s.image.loadimages(
+                path="images/",
+                caslib="dlib",
+                decode=True,
+                casout={'name' : 'imgsServer'}
+            )
+            
+            # Perform the same process on both client and server images
+            self.s.CASTable('imgsOutClient', replace=True)
+            self.s.CASTable('imgsOutServer', replace=True)
+            self.s.image.processimages(
+                table={'name': 'imgsClientDirectory'},
+                casout={'name': 'imgsOutClient', 'replace': True},
+                steps=[{'step':{'stepType':'RESIZE', 'type': 'BASIC', 'width': 200, 'height': 200}}]
+            )
+            self.s.image.processimages(
+                table={'name': 'imgsServer'},
+                casout={'name': 'imgsOutServer', 'replace': True},
+                steps=[{'step':{'stepType':'RESIZE', 'type': 'BASIC', 'width': 200, 'height': 200}}]
+            )
+
+            # Compare results to see if they are affected in the same way
+            length = len(self.s.fetchimages('imgsOutClient').Images.Image)
+            found = [[True] * length for x in range(length)]
+            imgs1 = self.s.fetchimages('imgsOutServer').Images.Image
+            imgs2 = self.s.fetchimages('imgsOutClient').Images.Image
+            for k in range(length):
+                for i in range(length):
+                    diff = ImageStat.Stat(ImageChops.difference(imgs1[k], imgs2[i])).mean
+                    for j in range(len(diff)):
+                        if diff[j] > validThreshold: found[k][i] = False
+            for i in found:
+                assert True in i
+            
+            # Assert that the table created from nonexistent path has no images
+            assert len(self.s.fetchimages('imgsClientEmpty').Images.Image) == 0
```

Comment:
This assert will need to be moved to the other test function. 
test_imagetable_load_client_images_invalid_path

Reply:
I've created the other test function and moved it over.